oncotator/input/MafliteInputMutationCreator.py

```diff
@@ -111,8 +117,11 @@ def __init__(self, filename, configFile='maflite_input.config', genomeBuild="hg1
                     if col != "build":
                         missingRequiredHeaders.append(col)
         missingRequiredHeaders.sort()
-        
-        self.logger.info("Initializing a maflite file with the following header: " + str(self._tsvReader.getFieldNames()))
+
+        if mutation_data_factory is None:
```

Comment:
Can this check be lifted into the superclass constructor?  It looks like it's repeated in a few places.